config/custom-environment-variables.yaml

```diff
@@ -14,9 +14,9 @@ services:
     scope: 'openid profile roles manage-user search-user'
     responseType: 'code'
 session:
-  secret: 'local'
   redis:
     host: REDIS_HOST
+    port: REDIS_PORT
```

Comment:
Does this new config need to be in default.yaml too?

Reply:
Fixed in https://github.com/hmcts/idam-user-dashboard/pull/108/commits/64b356222517afd091e37ed3d3a332a88fc7869d